app/js/handlebarsScoreTable.ts

```diff
@@ -214,7 +214,9 @@ function createUserResults(resultData, questionData): Object {
         userResultsTable[result] = {
             result: result,
             question: questionObj[result],
-            userAnswer: userResults[result].answerID
+            userAnswer: userResults[result].answerID,
+            notes: userResults[result].notes,
+            notesEmpty: userResults[result].notes.length > 1
```

Comment:
surely this should be `> 0` as someone could leave a 1 character note?

Reply:
also calling this property `notesEmpty` is the wrong way round, it should be called `hasNotes` or something